metabase/metabase
# First week of month not being visible in calendar selector

Closes https://github.com/metabase/metabase/issues/29692

### Description

When selecting Specific Date Filter the amount of days for a month don't add up. Some days from the beginning of the month are lost

### How to verify

1. Go to New -> Dashboard -> Add a Filter -> Time -> All Options -> Save
2. Click on the Filter  -> Specific Dates -> All days of current month should be visible

<img width="1172" alt="image" src="https://user-images.githubusercontent.com/110378427/228915627-2e7ae7bd-a367-4343-a0f9-97b6f9a156f4.png">

### Demo

https://user-images.githubusercontent.com/7983744/229145088-73c43b79-bb4a-4c32-8068-3641a9d79288.mov

### Checklist



- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/metabase/metabase/29722)
